<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stock</class>
 <widget class="QDialog" name="Stock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>New/Edit Product:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Remove Product:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Product Id:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stockButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>411</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Stock</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>65</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Id:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>148</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="stockName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="stockId"/>
    </item>
    <item>
     <widget class="QLineEdit" name="stockPrice"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addstockButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editstockButton">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>148</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="idstock"/>
    </item>
    <item>
     <widget class="QPushButton" name="removeButton_2">
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
